Dockerfile

```
@@ -13,8 +22,10 @@ RUN apk -U upgrade && apk add --no-cache \
     openssh-client \
     openssh-keygen \
     python3 \
-    tzdata
+    tzdata && \
+    /build/alpine-fips.sh
```

Just a small nitpick, but I think it would be easier to read and maintain if this fibs setup remains in its own layer, with the copies alongside to group the logic.

```suggestion
    tzdata

COPY scripts/alpine-fips.sh /build/
RUN /build/alpine-fips.sh
COPY openssl.cnf /etc/ssl/openssl.cnf
```